src/views/Releases/Release/index.jsx

```diff
@@ -90,10 +130,64 @@ export default function Release(props) {
     setReleaseEditorValue(value);
   };
 
-  // TODO: Add mutations
-  const handleReleaseCreate = () => {};
-  const handleReleaseUpdate = () => {};
-  const handleReleaseDelete = () => {};
+  const handleReleaseCreate = async () => {
+    // Newly created Releases never need signoff, so we can always
+    // safely create them directly instead of using Scheduled Changes.
+    const { error } = await createRel(
+      releaseNameValue,
+      productTextValue,
+      releaseEditorValue
+    );
+
+    if (!error) {
+      props.history.push(`/releases#${releaseNameValue}`);
+    }
+  };
+
+  const handleReleaseUpdate = async () => {
+    // Updates to Releases require signoff in some circumstances,
+    // but we don't have a need to schedule them for specific times.
+    // Because of this, we always schedule them for slightly in the future,
+    // which means that changes that don't require signoff will happen
+    // almost immediately, and changes that do require signoff will wait
+    // until those are completed.
```

Comment:
The comments have been really helpful :)

Reply:
Thanks! I hope my future self thinks so too :).